scripts/convert_openie_to_conll.py

```diff
@@ -245,7 +243,7 @@ def pad_line_to_ontonotes(line, domain) -> List[str]:
     word_ind, word = line[:2]
     pos = "XX"
     oie_tags = line[2:]
-    line_num = 0
+    line_num = "0"
```

Comment:
I probably wouldn't change this.  Not sure why mypy isn't complaining about it, but this doesn't seem like the right fix if there's a type complaint (unless you traced the logic and are *sure* that downstream stuff doesn't care that this is a string, then it at least deserves a comment).

Reply:
Why would mypy complain about it? I changed it because it's supposed to return strings (and it wasn't), and the callers convert everything to string, so it's like there's something odd here. But I changed it back.

Reply:
Mypy would complain because it's not returning a str, even though it says it is.  If the only caller is in this file, and it just converts everything to strings anyway, then sure, it's ok to remove it.  I was just looking at that change without context and thinking it was a scary change.

Reply:
Oh, okay. I changed it again.